data/categories.txt

```diff
@@ -8621,6 +8622,7 @@ uk:^Пункт постачання води
 vi:^Điểm cấp nước
 zh-Hans:^取水点
 zh-Hant:^飲水站
+fa:شیر آب
```

Comment:
https://wiki.openstreetmap.org/wiki/Tag:amenity=water_point
Not that it's "drinking water" not just any "water".
Is it true for what you suggest?